src/lib.rs

```diff
@@ -107,4 +118,24 @@ mod tests {
         assert_eq!(bank.rdh_version, Some("300".to_string()));
         assert_eq!(bank.pin_tan_version, Some("300".to_string()));
     }
+
+    // These tests use the real CSV input data.
+    // As such, the tests might break randomly if the input document changes.
+    // This should happen rarely enough that it's still worthwhile to test with real data, though.
+
+    #[test]
+    fn getting_random_bank_works() {
```

Comment:
Could you rename this to `get_bank_by_bank_code`?

Reply:
Then we would get a name collision, but I propose `get_bank_by_bank_code_test` so like with the `_test` suffix. Would that work for you?